lib/puppet-strings/describe.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+require 'puppet-strings/json'
+
+# The module for command line documentation related functionality.
+module PuppetStrings::Describe
+  # Renders requested types or a summarized list in the current YARD registry to STDOUT.
+  # @param [Array] describe_types The list of names of the types to be displayed.
+  # @param [bool] list Create the summarized list instead of describing each type.
+  # @param [bool] providers Show details of the providers.
+  # @return [void]
+  def self.describe(describe_types = [], list = false, providers = false)
```

Comment:
sort of a nit, but we've been calling the methods that create the final output `render`